app/components/selected_users/index.tsx

```diff
@@ -0,0 +1,240 @@
+    const keyboardHeight = useKeyboardHeight();
+    const insets = useSafeAreaInsets();
+
+    const scrollViewHeight = useSharedValue(0);
+    const [isVisible, setIsVisible] = useState(false);
+
+    useEffect(() => {
+        setIsVisible(Boolean(Object.keys(selectedIds).length));
+    }, [selectedIds]);
+
+    const handleOnPress = async () => {
+        onPress();
+    };
+
+    const users = useMemo(() => {
+        const u = [];
+        for (const id of Object.keys(selectedIds)) {
+            if (!selectedIds[id]) {
+                continue;
+            }
+
+            u.push(
+                <SelectedUser
+                    key={id}
+                    user={selectedIds[id]}
+                    teammateNameDisplay={teammateNameDisplay}
+                    onRemove={onRemove}
+                    testID='create_direct_message.selected_user'
+                />,
+            );
+        }
+        return u;
+    }, [selectedIds, teammateNameDisplay, onRemove]);
+
+    const marginBottom = useMemo(() => {
+        if (!keyboardHeight) {
+            return isTablet ? TABLET_MARGIN_BOTTOM : 0;
+        }
+
+        let margin = BOTTOM_TAB_HEIGHT + BUTTON_HEIGHT + insets.bottom;
+        margin += isTablet ? 0 : -10;
+        return margin;
+    }, [keyboardHeight]);
+
+    const totalPanelHeight = useDerivedValue(() => (
+        isVisible ? scrollViewHeight.value + BUTTON_HEIGHT : 0
+    ), [isVisible]);
+
+    const animatedViewStyle = useAnimatedStyle(() => {
+        return {
+            height: withTiming(totalPanelHeight.value, {duration: 200}),
+            marginBottom,
+        };
+    }, [marginBottom]);
+
+    const onLayout = useCallback((e: LayoutChangeEvent) => {
+        scrollViewHeight.value = Math.min(SCROLL_VIEW_MAX_HEIGHT, e.nativeEvent.layout.height);
+    }, []);
+
+    const contents = useMemo(() => (
+        <Animated.View style={[style.container, animatedViewStyle]}>
+            <ScrollView style={{height: scrollViewHeight.value}}>
+                <View
+                    style={style.users}
+                    onLayout={onLayout}
+                >
+                    {users}
+                </View>
+            </ScrollView>
+            <Button
+                onPress={handleOnPress}
+                icon={buttonIcon}
+                text={buttonText}
+            />
+        </Animated.View>
+    ), [users]);
+
+    const animatedToastStyle = useAnimatedStyle(() => {
+        return {
+            bottom: TOAST_BOTTOM_MARGIN +
+                totalPanelHeight.value +
+                marginBottom,
+            opacity: withTiming(showToast ? 1 : 0, {duration: 300}),
+            position: 'absolute',
+        };
+    }, [showToast, marginBottom]);
+
+    const toast = useMemo(() => (
+        <Toast
+            animatedStyle={animatedToastStyle}
+            iconName={toastIcon}
+            style={style.toast}
+            message={toastMessage}
+        />
+    ), [animatedToastStyle]);
```

Comment:
Same comment as the previous useMemo

Reply:
In this case, the Toast component requires an animated style to be passed in as a prop.  That's the reason it is wrapped in an animatedStyle.